web/app.js

```diff
@@ -1457,6 +1460,9 @@ function webViewerInitialized() {
     if ('useonlycsszoom' in hashParams) {
       PDFJS.useOnlyCssZoom = (hashParams['useonlycsszoom'] === 'true');
     }
+    if ('printautorotate' in hashParams) {
```

Comment:
Thank you!
Can you please also add a JSDoc comment for the new `PDFViewer` parameter at https://github.com/mozilla/pdf.js/blob/master/web/pdf_viewer.js#L69?